src/multiplayer-servers/getting-started/setup-third-party-oauth.md

```diff
@@ -0,0 +1,150 @@
+
+![Scopes Screenshot](images/authentication/create-oidc-provider-thirdstep-screenshot.png)
+
+ Common scopes include:
+
+| Scope         | Description                                                               |
+|---------------|---------------------------------------------------------------------------|
+| openid        | Required for OpenID Connect; requests and ID token                        |
+| email         | Requests the user's email address                                         |
+| profile       | Requests additional user profile information, such as name and picture    |
+| groups        | Requests group membership claims (if supported by the identity provider)  |
+| custom scopes | Applications or APIs can define their own scopes for specific permissions |
+
+OIDC Providers return **Claims** as a result of a successful authentication. These **Claims** denote information about the authenticated user.
+
+GameFabric uses these **Claims** to map to data related to the user, such as **UserID** and **UserName**
+
+When configuring outstanding providers, such as Microsoft ENTRA, these values are standard. Claim `sub` is mapped to **UserID** and `name` is mapped to **UserName**.
+
+But, it is still possible to use different **Claims** to be configured onto **UserID** and **UserName**.
+
+**UserID Key** text input allows an incoming **Claim** to be mapped onto **UserID** field and **UserName Key** text input allows an incoming **Claim** to be mapped onto **UserName** field.
+
+#### Prompt
+
+OIDC Providers execute authentication via displaying a prompt to the user. This behaviour of the provider may be configured via forwarding the **Prompt** parameter with the authentication request.
+
+Possible **Prompt** values are:
+
+| Prompt         | Description                                                                                                        |
+|----------------|--------------------------------------------------------------------------------------------------------------------|
+| none           | Ensures **no user interaction** occurs. If the user is not already authenticated, the request fails with an error. |
+| login          | Forces the user to **re-authenticate**, even if they are already logged in.                                        |
+| consent        | Forces the identity provider to show a consent screen, even if the user has already granted consent.               |
+| select_account | Prompts the user to choose an account if they are logged in with multiple accounts.                                |
+
+**Prompt** text input allows configuring this parameter, therefore specifying how the OIDC Provider should prompt the user during the authentication. GameFabric will forward this parameter to the provider. Default value will be `consent` if not configured otherwise.
```

Comment:
All of this info is just a repeat of what has been stated above, excluding the default. This is not a high school essay graded on the number of words, we should try not to repeat all info.

```suggestion
```